reviewer-2-comments.md

```diff
@@ -0,0 +1,13 @@
+# Reviewer 2
+## General comments
+I have reviewed your figure and I respect all the work that was put into it. 
+Incorporating a piece of fruit within a logo is certainly a novel approach.
+That said, I have some major concerns regarding the values that your peach conveys. 
+Thus, I have provided comments to help your peach better symbolize eternal youth and be more accessible for color-blind users.
+
+## Specific comments
+- Please change the color of the peach to [252,141,89]. Please refer to https://colorbrewer2.org for appropriate color combinations in the future.
+- Please change the color of the leaf to [145,207,96].
+- Please change all outline widths to 1.
+- Please decrease the ASB text width to 4.
```

Comment:
Specify that it's the asb text *linewidth* so there's no confusion about how tall/wide the text should be. I could see someone trying to set the whole logo width to 4 or something.